packages/amplication-data-service-generator/src/server/resource/dto/create-field-class-property.ts

```diff
@@ -340,7 +348,7 @@ function createGraphQLFieldType(
   entity: Entity,
   isQuery: boolean
 ): namedTypes.Identifier | namedTypes.ArrayExpression {
-  if (prismaField.isList) {
+  if (prismaField.isList && isToManyRelationField(field) && !isQuery) {
```

Comment:
This is incorrect, the change incorrectly exclude fields when isList && !isRelation

Reply:
we need that when it is prisma field list and to-many relation and not query that the type would not be inside an array,
otherwise, this condition is true. Then, on line 390 we have the following condition:

```
if (isToManyRelationField(field)) {
    return createEntityListRelationFilterID(prismaField.type);
  }
```

And as a result we got [EntityListRelationField]